src/creds/VerifiableCredential.js

```diff
@@ -476,17 +476,19 @@ function VerifiableCredentialBaseConstructor(identifier, issuer, expiryIn, ucas,
 
     // Test next level
     if (verifiedlevel === VERIFY_LEVELS.PROOFS
-        && hVerifyLevel >= VERIFY_LEVELS.ANCHOR
```

Comment:
Is it correct that verify(leve) can now verify everything up to Anchor but not including Anchor? If so this should be probably documented.

Reply:
Yes you are correct needs better documentation